Exception when calling basic_api->create: (400)
Reason: Bad Request
HTTP response headers: HTTPHeaderDict({'Date': 'Sun, 11 Dec 2022 17:14:11 GMT', 'Content-Type': 'application/json;charset=utf-8', 'Content-Length': '227', 'Connection': 'keep-alive', 'CF-Ray': '777fdaeb7eb8e788-MDE', 'Strict-Transport-Security': 'max-age=31536000; includeSubDomains; preload', 'Vary': 'origin, Accept-Encoding', 'CF-Cache-Status': 'DYNAMIC', 'Access-Control-Allow-Credentials': 'false', 'X-HubSpot-Correlation-Id': 'b7f72365-1e24-4667-a04d-b689e683a8f4', 'X-HubSpot-RateLimit-Daily': '500000', 'X-HubSpot-RateLimit-Daily-Remaining': '499997', 'X-HubSpot-RateLimit-Interval-Milliseconds': '10000', 'X-HubSpot-RateLimit-Max': '150', 'X-HubSpot-RateLimit-Remaining': '149', 'X-HubSpot-RateLimit-Secondly': '15', 'X-HubSpot-RateLimit-Secondly-Remaining': '14', 'X-Trace': '2B1C10B616731CA898BC1A35FEB6BC9908C97C1098000000000000000000', 'Report-To': '{"endpoints":[{"url":"https:\\/\\/a.nel.cloudflare.com\\/report\\/v3?s=YPrkS%2B7T12LKNBiAfysGUAsjrlq99Kz3mEUMu4cNC3xeTiZj1%2Bo8WQbLXYNPRXJzOpkGlWFZgRKvoDpTCcRYuFSSRuEpbRF0ku%2Bhs50uAU1uk8lrDd8A%2FXk97RRXI%2FD6"}],"group":"cf-nel","max_age":604800}', 'NEL': '{"success_fraction":0.01,"report_to":"cf-nel","max_age":604800}', 'Server': 'cloudflare', 'alt-svc': 'h3=":443"; ma=86400, h3-29=":443"; ma=86400'})
HTTP response body: {"status":"error","message":"Invalid input JSON on line 1, column 77: Cannot deserialize value of type `java.lang.String` from Array value (token `JsonToken.START_ARRAY`)","correlationId":"b7f72365-1e24-4667-a04d-b689e683a8f4"}